gh_actions_exporter/config.py

```diff
@@ -45,6 +45,12 @@ class Settings(BaseSettings):
     }
     flavor_label: Optional[str] = 'flavor'
     default_cost: Optional[float] = 0.008
+    github_app_private_pem: SecretStr # Checker
+    github_app_id: int
+    github_app_installation_id: int
+    title: str = "Workflow Costs"
```

Comment:
Both `title` and `summary` variables should be giving more context than just title as there might be multiple check-runs in the future with other type of stats, not cost related.

Reply:
Exactly, there will be other `check-runs`, so in other words, the `title` and `summary` here will have nothing to do with the others